augur/datasources/githubapi/routes.py

```diff
@@ -15,6 +15,78 @@ def create_routes(server):
     ### GROWTH, MATURITY, AND DECLINE ###
     #####################################
 
+    """
+    @api {get} /:owner/:repo/githubapi/issues/closed Closed Issues
```

Comment:
Please add `/timeseries/` to the route to maintain consistency across timeseries endpoint routes. My suggestion for the full route is `/:owner/:repo/timeseries/githubapi/issues/closed`.